src/Service/JsonService.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+
+namespace App\Service;
+
+use Symfony\Component\Serializer\SerializerInterface;
+
+class JsonService
+{
+    /**
+     * @var SerializerInterface
+     */
+    private $serializer;
+
+    /**
+     * JsonService constructor.
+     * @param SerializerInterface $serializer
+     */
+    public function __construct(SerializerInterface $serializer)
+    {
+        $this->serializer = $serializer;
+    }
+
+    /**
+     * @param $object
```

Comment:
```suggestion
     * @param object $object
```
At least PHPStorm will highlight parameter if it will be not object